style/base.css

```diff
@@ -231,3 +231,29 @@ ul.jp-notebook-job-output-formats-options {
 .jp-notebook-job-navigation-tab.active {
   border-block-end: 2px solid var(--jp-brand-color1);
 }
+
```

Comment:
Let's move the CSS for this component into a separate CSS file that is imported into base.css, following the practice in #13

Reply:
Done